packages/waku/src/router/create-pages.ts

```diff
@@ -175,7 +175,11 @@ export function createPages(
         page as {
           staticPaths: string[] | string[][];
         }
-      ).staticPaths.map((item) => (Array.isArray(item) ? item : [item]));
+      ).staticPaths.map((item) =>
+        (Array.isArray(item) ? item : [item]).map((i) =>
+          i.replace(/\./g, '').replace(/ /g, '-'),
+        ),
```

Comment:
Looking at the doc change makes me feel like maybe we can define `sanitizeSlug` function out, and do this?
```suggestion
        (Array.isArray(item) ? item : [item]).map(sanitizeSlug),
```